app/views/alaveteli_pro/account_request/index.html.erb

```diff
@@ -1,82 +1,135 @@
+      <% else %>
+        <h3><%= _("Now in beta") %></h2>
+        <a href="#input-beta" class="button" id="launch-access">
+          <%= _("Get pre-launch access") %>
+        </a>
+      <% end %>
+    </div>
   </div>
-
 </div>
 <div class="marketing__section">
   <div class="marketing__section__heading">
-    <h2>WhatDoTheyKnow<sup>Pro</sup></h2>
-    <p>An all-in-one FOI toolkit for journalists, activists and campaigners</p>
+    <h2><%= AlaveteliConfiguration.pro_site_name %></h2>
+    <p><%= _('An all-in-one FOI toolkit for journalists, activists and ' \
+             'campaigners') %></p>
   </div>
   <div class="marketing__section__content">
     <div class="marketing__section__content-items">
       <ul class="marketing__section__feature-list">
-        <li>Keep <strong class="marketing-highlight">requests and responses private</strong> while you work on your story</li>
-        <li>A powerful <strong class="marketing-highlight">private dashboard</strong>: track and manage your FOI projects</li>
-        <li>A super-smart <strong class="marketing-highlight">to-do list</strong>: follow the progress of your requests</li>
-        <li><strong class="marketing-highlight">Action alerts</strong>: know when it’s time to take the next step</li>
-        <li><strong class="marketing-highlight">Save as draft</strong> to get back to it later</li>
+        <li>
+          <%=_('Keep requests and responses private while you work on your ' \
+               'story') %>
+        </li>
+        <li>
+          <%=_('A powerful private dashboard: track and manage your FOI ' \
+               'projects') %>
+        </li>
+        <li>
+          <%=_('A super-smart to-do list: follow the progress of your ' \
+               'requests') %>
+        </li>
+        <li>
+          <%=_('Action alerts: know when it’s time to take the next step') %>
+        </li>
+        <li>
+          <%=_('Save as draft to get back to it later') %>
+        </li>
       </ul>
 
       <div class="side-by-side-elements screenshots">
         <div class="side-by-side-element">
-          <a href="<%= image_path("alaveteli-pro/screenshot-dashboard.jpg") %>"><%= image_tag("alaveteli-pro/screenshot-dashboard.jpg", :alt => "Pro brings a new dashboard view") %></a>
+          <a href="<%= image_path("alaveteli-pro/screenshot-dashboard.jpg") %>">
+            <%= image_tag("alaveteli-pro/screenshot-dashboard.jpg",
+                          :alt => "Pro brings a new dashboard view") %>
+          </a>
         </div>
         <div class="side-by-side-element">
-          <a href="<%= image_path("alaveteli-pro/screenshot-requests.jpg") %>"><%= image_tag("alaveteli-pro/screenshot-requests.jpg", :alt => "Pro features a new request list view") %></a>
+          <a href="<%= image_path("alaveteli-pro/screenshot-requests.jpg") %>">
+            <%= image_tag("alaveteli-pro/screenshot-requests.jpg",
+                          :alt => "Pro features a new request list view") %>
```

Comment:
We should probably make up the alt text for translation as well (we already do this in a few places)